packages/wagtail/modules/providers/PageLayoutProvider/types.ts

```
@@ -0,0 +1,15 @@
+export type PageLayoutProviderProps = {
+  children: React.ReactNode
+  customLayoutSettings?: Partial<PageLayoutContextStates>
+}
+
+export type PageLayoutContextStates = {
+  loadContainer: Boolean
```

Biome

[error] Don't use 'Boolean' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'boolean' instead


(lint/complexity/noBannedTypes)